checks.md

```diff
@@ -161,3 +161,12 @@ This check tries to determine if the project has branch protection enabled.
   force pushes or deletion of your important branches.
 - For GitHub, check out the steps
   [here](https://docs.github.com/en/github/administering-a-repository/managing-a-branch-protection-rule).
+
+
+## Automatic-Dependency-Update
+
+This check tries to determine if the project has dependencies automatically updated.
+
+**Remediation steps**:
+- Enable dependabot for the project and have a dependabot.yml file in the repository. 
```

Comment:
s/and have a dependabot.yml file in the repository/by placing a .github/dependabot.yml in the root of your repository. See https://docs.github.com/en/code-security/supply-chain-security/enabling-and-disabling-version-updates for more information